header.php

```diff
@@ -10,21 +10,19 @@
 
 		<script type="text/javascript">
 			var _sf_startpt = (new Date()).getTime();
-			<?php if(GA_ACCOUNT):?>
 
-			var GA_ACCOUNT  = '<?=GA_ACCOUNT?>';
-			var _gaq        = _gaq || [];
-			_gaq.push(['_setAccount', GA_ACCOUNT]);
-			_gaq.push(['_setDomainName', 'none']);
-			_gaq.push(['_setAllowLinker', true]);
-			_gaq.push(['_trackPageview']);
+			<?php if(GA_ACCOUNT):?>
+			<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Need to remove inline `<script>` tags, since this snippet is already wrapped in a `<script>` (see line 11)